### PR TITLE
Completed css-positioning exercise - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**
Screenshot 2023-01-27 at 9 26 45 AM
Screenshot 2023-01-27 at 9 39 16 AM
Screenshot 2023-01-27 at 9 41 25 AM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: relative;
+  top: 175px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
```
Comment (JenBowman1) [inline css-positioning/absolute.css:33]: please use the position: absolute property for the overlay as well as the top, right, bottom, and left properties to make sure the gradient goes over the image!

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
Hi Justin, one small change

File: css-positioning/absolute.css (hunk 1 above)
Comment (justinsong1608) [reply css-positioning/absolute.css:33]: Screenshot 2023-01-27 at 10 10 11 AM

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**